### PR TITLE
Add context menus

### PR DESCRIPTION
**Pull request by yisibl**
* Copy
* Select All
* Reload
* Inspect Element(Open Developer Tools)

Now we can directly use context menus.

### CODE REVIEW

**Comment by yisibl** [comment]
@maxkueng

File: client.js [hunk 1]
```diff
@@ -24,50 +29,140 @@ window.addEventListener('keydown', function (ev) {
...
+        return 'Ctrl+Shift+I'
+      }
+    })(),
+    click: function (item, focusedWindow) {
+      if (focusedWindow) {
+        focusedWindow.toggleDevTools()
+      }
+    }
+  }]
+}]
```
Comment (maxkueng) [inline client.js]: Was there a particular reason why you changed the formatting of the menu template? In my opinion it doesn't improve readability or bring other benefits.

File: client.js [hunk 2]
```diff
@@ -24,50 +29,140 @@ window.addEventListener('keydown', function (ev) {
...
-  vmdSubmenu = [
-    { label: 'About vmd', selector: 'orderFrontStandardAboutPanel:' },
-    { type: 'separator' }
-  ].concat(vmdSubmenu)
+  vmdSubmenu = [{
+    label: 'About vmd',
+    selector: 'orderFrontStandardAboutPanel:'
+  }, {
+    type: 'separator'
+  }].concat(vmdSubmenu)
```
Comment (maxkueng) [inline client.js]: Same here

File: client.js (hunk 1 above)
Comment (yisibl) [inline client.js]: My editor formatting tool is automatically formatted.

**Comment by maxkueng** [comment]
Great feature! I've added some notes regarding formatting.

File: client.js (hunk 1 above)
Comment (yisibl) [inline client.js]: Standard doesn't seem to have such a rule?

Comment (maxkueng) [inline client.js]: If there was no particular reason for the reformatting ("my editor does it" doesn't count ;) ) it'd be cool if the formatting was left intact. However, if you think it makes the code better we can consider changing it.

Yeah, `standard` doesn't cover everything. If we wanted automatic formatting we'd have to use something like [jsfmt](https://github.com/rdio/jsfmt).

Comment (mattdesl) [inline client.js]: @maxkueng There is also `standard-format` module. Personally I just run most of my code through it these days, tho I agree it should not be part of this PR.

Comment (yisibl) [inline client.js]: @maxkueng I used https://packagecontrol.io/packages/HTML-CSS-JS%20Prettify

I think this format will produce better diff.

Comment (maxkueng) [inline client.js]: @yisibl Yeah, I agree that writing objects on multiple lines is better and produces better diffs and it makes a difference especially when objects have more than one or two keys.  
But my question about the indentation and formatting was more about why this:

```js
var obj = [{
  k1: 'v1',
  k2: 'v2'
}, {
  k1: 'v1',
  k2: 'v2'
}]
```

over this:

```js
var obj = [
  {
    k1: 'v1',
    k2: 'v2'
  },
  {
    k1: 'v1',
    k2: 'v2'
  }
]
```

?

Comment (yisibl) [inline client.js]: Which style do you want to use? I can adjust it. And then again push. I want to merge the PR as soon as possible.

Comment (maxkueng) [inline client.js]: @yisibl I've created a pull request against your branch that changes the array indentation back to how it was. If you think it looks okay, please merge it.

**Comment by yisibl** [comment]
@maxkueng Thanks, done!

**Comment by maxkueng** [comment]
Thanks! Great feature! :shipit: vmd@1.16.0 is on npm
